appsec-kit-starter/src/main/java/com/vaadin/appsec/service/AppSecDevToolsPlugin.java

```diff
@@ -29,7 +29,7 @@ public class AppSecDevToolsPlugin implements DevToolsMessageHandler {
     private static final Logger LOGGER = LoggerFactory
             .getLogger(AppSecDevToolsPlugin.class);
 
-    private final Map<DevToolsInterface, Registration> scanEventRegistrations = new HashMap<>();
+    protected final Map<DevToolsInterface, Registration> scanEventRegistrations = new HashMap<>();
```

Comment:
We should not need this: tests should not need to access private fields, that means they're testing that the code is written in a certain way and possibly not that it behaves as expected.

Also, let's make this a `ConcurrentHashMap` to avoid access exceptions when handling connection from different threads:
```suggestion
    private final Map<DevToolsInterface, Registration> scanEventRegistrations = new ConcurrentHashMap<>();
```

Reply:
Reverted this change and it is now a `ConcurrentHashMap`.